tests/test_sphere.py

```diff
@@ -9,7 +8,7 @@ def sphere_cube(tmp_path: Path):
     cube = Print3D(x_size=11, y_size=11, z_size=11, part_no="testing_sphere")
 
     for side in (cube.left, cube.right, cube.top, cube.bottom, cube.front, cube.back):
-        side.sphere(pos=[7, 7], diameter=6, sink=1)
+        side.sphere(pos=(7.0, 7.0), diameter=6, sink=1)
```

Comment:
This now breaks the test that check the file hash, seems the test is broken not the code.

Reply:
See https://github.com/tsolo-io/cycax/pull/47/commits/46fb355c6686a3bd9be5ff074e05a4f4da7dc848 where I removed the flaky tests.